typing/env.mli

```diff
@@ -26,6 +26,7 @@ type summary =
   | Env_type of summary * Ident.t * type_declaration
   | Env_extension of summary * Ident.t * extension_constructor
   | Env_module of summary * Ident.t * module_declaration
+
```

Comment:
You probably didn't intend to commit that.

Reply:
Thanks